### PR TITLE
Add ZMQ multiple DPU support. 

### PR DESCRIPTION
**Pull request by liuh-80**
Add ZMQ multiple DPU support. 

#### Why I did it
Support ZMQ connection to multiple DPU.

#### How I did it
Add ZMQ multiple DPU support. 

#### How to verify it
Manually test.
Add new UT.

#### Work item tracking
Microsoft ADO (number only): 26731929

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

#### Description for the changelog
Add ZMQ multiple DPU support.

### CODE REVIEW

**Comment by qiluo-msft** [comment]
Is this PR unblocked?

**Comment by liuh-80** [comment]
> Is this PR unblocked?

Working on change to subscribe CONFIG_DB change. SWIG go wrapper have some limitation, will update this and sonic-swss-common PR later.

**Comment by liuh-80** [comment]
This PR depends on https://github.com/sonic-net/sonic-gnmi/pull/194 merge first.

File: sonic_data_client/mixed_db_client.go [hunk 1]
```diff
@@ -160,6 +159,17 @@ func getZmqClientByAddress(zmqAddress string) (swsscommon.ZmqClient, error) {
 	return client, nil
 }
 
+func removeZmqClient(zmqClient swsscommon.ZmqClient) (error) {
+	for address, client := range zmqClientMap {
+		if client == zmqClient {
```
Comment (qiluo-msft) [inline sonic_data_client/mixed_db_client.go:164]: >zmqClient [](http://example.com/codeflow?start=15&length=9)

we need to handle resource dispose aggressively. not a blocking issue right now.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**